plugins/sonarqube/api-report.md

```diff
@@ -21,17 +21,34 @@ export const EntitySonarQubeCard: (props: {
   duplicationRatings?: DuplicationRating[] | undefined;
 }) => JSX.Element;
 
+// @public (undocumented)
+export const EntitySonarQubeContentPage: (
+  props: SonarQubeContentPageProps,
+) => JSX.Element;
+
 // @public (undocumented)
 export const isSonarQubeAvailable: (entity: Entity) => boolean;
 
+// @public (undocumented)
+export const SONARQUBE_PROJECT_KEY_ANNOTATION = 'sonarqube.org/project-key';
+
 // @public (undocumented)
 export const SonarQubeCard: (props: {
   variant?: InfoCardVariants;
   duplicationRatings?: DuplicationRating[];
 }) => JSX.Element;
 
 // @public (undocumented)
-const sonarQubePlugin: BackstagePlugin<{}, {}, {}>;
-export { sonarQubePlugin as plugin };
-export { sonarQubePlugin };
```

Comment:
Oh, note how this one changed. Used to be two exports but now it's only one (at the bottom of this file). The reason it used to be two of them, is that the `plugin` shorthand was kept for backward compatibility but the `sonarQubePlugin` name is what people should be using.

I vote to keep this as-is, because this is a removal that we want to do anyway, and since we are making a breaking (minor) changeset anyway...

So my ask is, please add some text to the changeset: ```Removed the deprecated `plugin` export; please use `sonarQubePlugin` instead```.

Reply:
done